.github/workflows/eval-notebooks.yml

```diff
@@ -0,0 +1,51 @@
+name: Eval notebooks 
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
What does it mean that the workflow is triggered only for the main branch? When creating a PR that includes a notebook change, will the notebook be evaluated?

Reply:
Got an affirmative answer.